### ISSUE TITLE
Using print() in while loop to track values causes connection with TechnicHub to fail

### ISSUE THREAD

**Issue by AlbanT** [pybricks/support#36]
I uploaded the demo program (for the record, the first time I tried it pybricks complained that my motors weren't connected to the hub, after reflashing it worked):

    # Initialize hub and sensor
    hub = CPlusHub()
    
    # We'll use two motors. One is a dial to
    # set the speed of the other motor.
    motor = Motor(Port.C)
    dial = Motor(Port.A)
    
    # Say hello :)
    print("Hello, Pybricks!")

    # First, we'll move the dial to absolute zero.
    dial.run_target(500, 0, Stop.COAST)
    
    while True:
          # Set the speed based on dial angle
         speed = dial.angle()*3
    
         if abs(speed) < 100:
             speed = 0
         motor.run(speed)
The single `print("Hello, Pybricks!")` works fine. But I tried to print the value of speed inside the while loop:

    while True:
          # Set the speed based on dial angle
         speed = dial.angle()*3
    
         if abs(speed) < 100:
             speed = 0
         print(speed)
         motor.run(speed)

This also works but when I press the stop button the hub doesn't stop and both the start and stop button are greyed out. I have to remove the batteries and reconnect to take control again.

I suspect the loop is too fast, when I insert a `wait(10` to reduce the speed of the loop it works fine:

    while True:
          # Set the speed based on dial angle
         speed = dial.angle()*3
    
         if abs(speed) < 100:
             speed = 0
         wait(10)
         print(speed)
         motor.run(speed)

**Comment by dlech** [comment]
I am not able to reproduce the problem with the following script:

```python
from pybricks.pupdevices import Motor, ColorDistanceSensor
from pybricks.parameters import Port, Stop
from pybricks.tools import wait

motor = Motor(Port.D)
dial = Motor(Port.C)

while True:
      # Set the speed based on dial angle
     speed = dial.angle() * 3

     if abs(speed) < 100:
         speed = 0
     print(speed)
     motor.run(speed)
```

**Comment by kueden** [comment]
Yes, this is exactly the same issue. The motors keep on running, but there is no way to stop it with the stop button.

**Comment by dlech** [comment]
There have been significant changed made to Bluetooth in firmware v3.0.0b3. Can you please upgrade the firmware and test again?

**Comment by kueden** [comment]
David
I got it to work, but there is another issue. There is a bug with the v3.0.0b3. I can connect via blootooth. But the play bottun (F5 for download and run) does not light up. I played around and got it to work. My original issue is resolved. However I cannot reproducibly get software to run since the play button does not show up. Eventually things run. I also learned playing with it that the "getchar" that I use in all my software (issue #154) is not available in this new version 

So it looks like my minor issue is resolved, but I cannot run my software any more due to the not active play buttion and the missing getchar function. Now it is very probable that this is my lack of competence. I tried on another computer. Same thing.

Thanks for all your help

Markus
